config/locales/candidate_interface/english_foreign_language.yml

```
@@ -38,4 +45,8 @@ en:
               not_a_number: Enter your total score (this should be a number)
             award_year:
               blank: Enter year the assessment was done
-              future: Assessment year must be this year or a previous year
+              invalid_year: Enter a valid award year
+              not_an_integer: Award year must be an integer
+              not_a_number: Enter a valid award year
```

Borrowing error message language from the [Date input component](https://design-system.service.gov.uk/components/date-input/#error-messages):

```suggestion
              invalid_year: Assessment year must be a real year
              not_an_integer: Assessment year must be a real year
              not_a_number: Assessment year must be a real year
```